core-library/src/main/java/org/silverpeas/core/admin/space/dao/SpaceDAO.java

```diff
@@ -41,7 +41,7 @@
 public class SpaceDAO {
 
   private static final String SPACE_COLUMNS =
-      "id,domainFatherId,name,description,createdBy,firstPageType,firstPageExtraParam,orderNum,createTime,updateTime,removeTime,spaceStatus,updatedBy,removedBy,lang,isInheritanceBlocked,look,displaySpaceFirst,isPersonal";
+      "id,domainFatherId,name,description,spaceStatus,createdBy,firstPageType,firstPageExtraParam,orderNum,createTime,updateTime,removeTime,updatedBy,removedBy,lang,isInheritanceBlocked,look,displaySpaceFirst,isPersonal";
```

Comment:
As I tried your PR to understand something I was missing about the changes in AjaxServletLookV5, I took the opportunity to analyse all code changes with the IDE.
And I discovered that this code is never called into Silverpeas!
Do you agree with my observation?